main.py

```diff
@@ -1,10 +1,48 @@
-import init_django_orm  # noqa: F401
+import json
 
+import init_django_orm  # noqa: F401
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+    for player_details in data.items():
+        current_guild = player_details[1]["guild"]
+        race_details = player_details[1]["race"]
+        skill_details = race_details["skills"]
+
+        if not Race.objects.filter(name=race_details["name"]).exists():
+            Race.objects.create(
+                name=race_details["name"],
+                description=race_details["description"]
+            )
+        current_race = Race.objects.get(name=race_details["name"]).id
```

Comment:
use [get_or_create](https://docs.djangoproject.com/en/4.2/ref/models/querysets/#get-or-create) in this case